src/main/java/io/jenkins/plugins/credentials/secretsmanager/config/Client.java

```diff
@@ -77,6 +80,17 @@ public AWSSecretsManager build() {
         if (region != null && !region.isEmpty()) {
             builder.setRegion(region);
         }
+        
+        ProxyConfiguration proxyConfiguration = Jenkins.get().proxy;
```

Comment:
If I recall correctly, calling Jenkins.get() might either yield a null, or yield an exception, under certain circumstances. You'll need to wrap that in a defensive guard so that if the call doesn't work out, the code can just skip this block which constructs the proxy client configuration.

Reply:
I have added a try catch around that code.